packages/test/src/activities/interceptors.ts

```diff
@@ -1,9 +1,23 @@
 import * as activity from '@temporalio/activity';
 import { ConnectionLike } from '@temporalio/client';
 import { defaultDataConverter, LoadedDataConverter } from '@temporalio/common';
-import { ActivityExecuteInput, ActivityInboundCallsInterceptor, Next } from '@temporalio/worker';
+import {
+  ActivityExecuteInput,
+  ActivityInboundCallsInterceptor,
+  ActivityInterceptorsFactory,
+  Next,
+} from '@temporalio/worker';
 
 export class ConnectionInjectorInterceptor implements ActivityInboundCallsInterceptor {
+  public static asFactory(
```

Comment:
Don't love that we have this here but I don't mind since this is internal.

```suggestion
  public static createFactory(
```